anta/tests/routing/bgp.py

```diff
@@ -144,6 +144,55 @@ def _add_bgp_routes_failure(
     return failure_routes
 
 
+def _check_peer_capabilities(bgp_output: dict[str, Any], capabilities: list[MultiProtocolCaps], *, strict: bool = False) -> dict[str, Any]:
+    """Check for issues in BGP output for multiprotocol capabilities.
+
+    - Checks the only mentioned capabilities should be listed when strict is True.
+    - Checks the any capability(s) are missing from output.
+    - Checks the any capability(s) are not advertised, received, or enabled
+
+    Parameters
+    ----------
+        bgp_output: The BGP output from the device.
+        capabilities: List of multiprotocol capabilities to be verified.
+        strict: Optional check to verifies the only mentioned capabilities should be listed, otherwise test should fail. It defaults to false
+
+        The * indicates that all arguments following it must be provided as keyword arguments. To compensate boolean-type-hint-positional-argument (FBT001).
+
```

Comment:
Not required. We use this everywhere because of Ruff so that's fine.

Reply:
Addressed